ui/partials/perf/comparerror.html

```diff
@@ -0,0 +1,2 @@
+<div class="alert alert-danger" role="alert" ng-repeat="error in errors"><span class="fa fa-warning"></span> {{error}}</div>
+<a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
Just crossed my mind, if we wanted to have the 027bf5a.. sha's be uniform length so the resulting columns are also even, we could apply the same monospace property we use on the `.revision-holder` class in treeherder

`font: 12px "Lucida Console",Monaco,monospace;`